### ISSUE TITLE
Suggestion about recipes

### ISSUE THREAD

**Issue by Antifluxfield** [Blood-Asp/GT5-Unofficial#781]
1. Graphite for neutron reflectors (Inspired from #743 )
We do use it IRL (https://en.wikipedia.org/wiki/Neutron_reflector). And seems that the IC2 recipe we currently use is also trying to use graphite, but finally use the coal dust as an alterative since IC doesn't have it.
2. Higher tier components for higher tier fusion computer
Now all the 3 tiers of fusion computers are using IV field generator. Why not use those LuV or ZPMV tier components?

**Comment by Timeslice42** [comment]
These both seem like great ideas. More uses for graphite, and we get rid of coal in the reflector (which was just weird).

**Comment by Blood-Asp** [comment]
The current amount on graphite is likely not enough to build enough reflectors. So simple replacement would be unbalanced. Maybe a bit changed recipe too or at last some calculating how it would fit in.

The higher tier components are planned anyways. Fusion reactor component recipes will be moved partly into assembly line too. But first is the planned changes for circuits so i do not have to change that stuff twice.

**Comment by Antifluxfield** [comment]
> The material may be graphite, beryllium, steel, tungsten carbide, or other materials.

Perhaps we can use some of those rare materials, like Be and WC, using their plates or thick plates for an alternative recipe, reducing the graphite consumption.

**Comment by Antifluxfield** [comment]
Firstly I made a simple estimate of the graphite consumption:
Suppose the vein distribute in an area of 16*16, 6 layers of graphite ores, with a density of 20%, then each vien will contain about 300 graphite ores.
Meanwhile, each MarkII Fusion requires 32 fusion blocks, in other words, 64 Iridium Neutron Reflectors.
Suppose we hope that one or two diamond vein will provide enough graphite for a MarkII Fusion, then the graphite consumption of each INR should be about 5~10 graphite ores.

Accordingly, I made 3 plans for the graphite recipes of the reflectors. The first one can reduce the graphite cost, and the last two plans can increase the productive capability of graphite. Some of their combination should be a good way for balancing.
1. Alternative recipes
![2016-12-17_16 02 56](https://cloud.githubusercontent.com/assets/17354844/21285809/f984b40c-c47d-11e6-8ada-383bd88d6983.png)
![2016-12-17_16 02 54](https://cloud.githubusercontent.com/assets/17354844/21285678/67e829a0-c47a-11e6-9740-f5474f0d1192.png)
![2016-12-17_16 02 51](https://cloud.githubusercontent.com/assets/17354844/21285679/6c547bc4-c47a-11e6-9767-846237275f54.png)
The regular recipes are almost the same as the old recipes, consuming 128 tin foils, 128 graphite, 64 steel, 16 beryllium and 1 reiforced iridium plate for each INR; 
And the alternative recipes will reduce the cost of graphite down to 24 dust, meanwhile increasing the cost of other material, to 192 tin foils, 24 beryllium, 8 tungsten carbide and 1 reinforced iridium plate.
2. Raise the multiplier of graphite production
Currently, with each graphite ore producing slightly more then 2 graphite, we need about 12 ores for each INR by using the alternative recipes. Raising the multiplier to 2 or 3 will decrease that amount to 6 or 4 ores, respectively.
3. Just add another way to produce graphite
For example, process several(maybe 3) carbon dust and a few(maybe 200 or one 250) mB of helium(or other simillar thing) in the EBF, producing one graphite dust and two tiny piles of dark ash, making the graphite highly reproducible.
@Blood-Asp
